src/components/Navbar.js

```diff
@@ -80,7 +80,7 @@ function Navbar() {
                     className='btn-link'
                     onClick={closeMobileMenu}
                   >
-                    <Button buttonStyle='btn--outline' buttonSize='btn--mobile'>
+                    <Button buttonStyle='btn--outline' buttonSize='btn--medium'>
```

Comment:
The btn--medium font-size was 16px I think, which is more reasonable. I switched the class to that.